core/src/main/java/com/orange/ouds/core/component/OudsRadioButtonItem.kt

```diff
@@ -39,14 +39,17 @@ import com.orange.ouds.foundation.extensions.orElse
 /**
  * <a href="https://unified-design-system.orange.com/472794e18/p/90c467-radio-button" class="external" target="_blank">OUDS Radio button design guidelines</a>
  *
- * An OUDS radio button item is a layout containing an [OudsRadioButton], an associated label and several other optional elements.
- * It can be used in a list as a list item.
- * By clicking on a radio button item, the user changes the selected state of its radio button.
+ * Radio buttons are input controls that allow users to select a single option from a set of mutually exclusive choices.
+ *
+ * The **radio button item variant** can function as a simple input with a label in a selection group, or it can be combined with optional elements such as
+ * helper text, a divider, or an icon, allowing it to suit various use cases.
```

Comment:
Should we mention the additional text? I am not sure because this is not an exhaustive list.